CHANGELOG.md

```diff
@@ -5,31 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## \[Unreleased\]
+<!--
+  Developers: this project uses scriv (<https://scriv.readthedocs.io/en/stable/index.html>)
+  to maintain the changelog. To add an entry, create a fragment:
 
-### Added
-
-- The latest comment displayed in issues sidebar (<https://github.com/opencv/cvat/pull/6937>)
```

Comment:
@nmanovic You did not move these new entries into `legacy.md`. Please don't merge this until I can fix it.